src/components/helpwin/HelpWin.vue

```diff
@@ -29,18 +32,26 @@
   export default {
     props: {
       color: {type: String, required: false, default: 'red darken-3'},
-      icon: {type: String, required: false, default: 'help'},
-      title: {type: String, required: false, default: 'About'},
-      headline: {type: String, required: false, default: 'About Wegue'},
-      content: {type: String, required: false, default: '<h3>WebGIS with OpenLayers and Vue.js</h3> Template and re-usable components for webmapping applications with OpenLayers and Vue.js'},
-      infoLink: {type: String, required: false, default: 'https://github.com/meggsimum/wegue'},
-      infoLinkText: {type: String, required: false, default: 'More info'}
+      icon: {type: String, required: false, default: 'help'}
     },
     data () {
       return {
-        show: false
+        show: false,
+        title: '',
```

Comment:
I think the apply of the config data can be done directly `title: this.$appConfig.modules['wgu-helpwin'].windowTitle`, so implementing the `mounted` function could be obsolete.

Reply:
that's right. 
I added directly assigned the configs in the  `data` property